runtime/near-vm-runner/src/tests/compile_errors.rs

```diff
@@ -338,7 +379,7 @@ fn test_limit_locals_global() {
             .make(),
         )
         .expect(expect![[r#"
-            VMOutcome: balance 4 storage_usage 12 return data None burnt gas 139269213 used gas 139269213
+            VMOutcome: balance 4 storage_usage 12 return data None burnt gas 13001413761 used gas 13001413761
```

Comment:
Wow that's a lot of extra gas! Close to 100x more burnt.
I assume that's expected with the finite wasm changes? At least 15625 locals and the extra burning of 12 Ggas seems to check out if we charge the regular wasm op cost (0.82 Mgas) for each local.